demo/kserve/scripts/install/kserve-install.sh

```diff
@@ -86,59 +87,91 @@ mkdir ${BASE_CERT_DIR}
 # cd caikit-tgis-serving/demo/kserve
 
 # Install Service Mesh operators
```

Comment:
I can't comment at the right place, but related to this syntax:
```
if [ "$input" = "y" ]; then
   ....
else
    echo "ERROR: Please check the configmap and execute this script again"
    exit 1
fi
```
please have a look at the `guard clause` pattern (random [internet link](https://deviq.com/design-patterns/guard-clause) on the topic), it makes the code easier to read/write/extend:
```
if [ "$input" != "y" ]; then
    echo "ERROR: Please check the configmap and execute this script again"
    exit 1
fi
```